src/synthesizer_grids/cloudy/params/test_suite/alpha.yaml

```
@@ -1 +1 @@
-alpha: [-0.2, 0.0, 0.2, 0.4, 0.6, 0.8, 1.0]
\ No newline at end of file
+alpha_enhancement: [-0.2, 0.0, 0.2, 0.4, 0.6, 0.8, 1.0]
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)